src/flavors/williams/static/js/views/app-view.js

```diff
@@ -165,12 +171,16 @@ module.exports = AppView.extend({
     }).render();
 
     ReactDOM.render(
-      <UserMenu
-        router={this.options.router}
-        apiRoot={storeState.config.app.api_root}
-        currentUser={Shareabouts.bootstrapped.currentUser}
-        datasetDownloadConfig={storeState.config.app.dataset_download}
-      />,
+      <ThemeProvider theme={theme}>
+        <ThemeProvider theme={adjustedTheme}>
```

Comment:
Only the two `ThemeProvider` components were added here.

This diff might be easier to see by turning off the whitespace diff: https://css-tricks.com/use-button-element/#article-header-id-2